docs/concepts/index.md

```diff
@@ -1,6 +1,62 @@
-# Getting Started
+# ArcGIS Runtime toolkit for .NET
 
-Read more about the included controls here:
+The ArcGIS Runtime SDK for .NET toolkit contains controls and utilities you can use with [ArcGIS Runtime SDK for .NET](http://links.esri.com/dotnetsdk).
 
- - [Bookmarks View](../bookmarks-view.md)
- - [Augmented Reality View](../ar.md)
\ No newline at end of file
+There are two ways to use toolkit and AR toolkit in your apps:
+
+1. [Install from Nuget](xref:84935635-E77D-4243-90D0-8721DE8D39B8) - the fastest way to get toolkit into your app
+2. [Build from source](xref:3CCE1627-57F9-4ECB-A650-6D4D2E343337) - do this if you want to customize toolkit
+   
+## Features
+
+> See [the doc](xref:65F8FFAC-CD1F-4304-BDFF-423936F0B08A) for a full list of controls with screenshots
+
+- [ARSceneView](xref:D07D974D-8587-428E-BC75-EF531108515B): Part of the AR Toolkit, enables integration of GIS content and ARKit/ARCore.
+- [Bookmarks](xref:B1CCBC8A-91A2-485A-97E0-CF2B9F692B26): Shows bookmarks, from a map, scene, or a list; navigates the associated MapView/SceneView when a bookmark is selected.
+- **Compass**: Shows a compass direction when the map is rotated. Auto-hides when the map points north up.
+- **FeatureDataField**: Displays and optionally allows editing of a single field attribute of a feature.
+- **Legend**: Displays a legend for a single layer in your map (and optionally for its sub layers).
+- **MeasureToolbar**: Allows measurement of distances and areas on the map view.
+- **PopupViewer**: Display details and media, edit attributes, geometry and related records, and manage the attachments of features and graphics (popups are defined in the popup property of features and graphics).
+- **ScaleLine**: Displays current scale reference.
+- **SymbolDisplay**: Renders a symbol in a control.
+- **TimeSlider**: Allows interactively defining a temporal range (i.e. time extent) and animating time moving forward or backward.  Can be used to manipulate the time extent in a MapView or SceneView.
+
+### Features in Preview
+
+- **ChallengeHandler**: Displays SignInForm when accessing secure ArcGIS resources, as well as helper classes for storing credentials in Windows' credentials cache.
+- **SignInForm**: Displays a UI dialog to enter or select credentials to use when accessing secure ArcGIS resources.
+
+## Resources
+
+- [Documentation](https://esri.github.io/arcgis-toolkit-dotnet/)
+- [List of controls](xref:65F8FFAC-CD1F-4304-BDFF-423936F0B08A)
+- [Build the toolkit](xref:3CCE1627-57F9-4ECB-A650-6D4D2E343337)
```

Comment:
Why not use the relative paths instead of uid ? That way it works even on the github site when browsing source

Reply:
Because the rendered output is .html, I wasn't sure the links would still work if .md is specified, but it looks like docfx thought about that already. 

One advantage of the guid reference is the files can be moved around without breaking links, but I don't think that's worth the cost of them not working on GitHub/other doc systems.

Reply:
relative links to .md files will get updated to .html